pyro/infer/abstract_infer.py

```diff
@@ -285,23 +285,36 @@ class TracePredictive(TracePosterior):
     :param TracePosterior posterior: trace posterior instance holding
         samples from the model's approximate posterior.
     :param int num_samples: number of samples to generate.
+    :param (:obj:`list(str)`, optional) resample_sites: list of sites to resample from posterior.
+    All sites are resampled by default.
```

Comment:
I think you'll need to indent this line for sphinx to build correctly. You can run `make docs` locally to see if it works fine.